docs/winc-prerequisites.md

```diff
@@ -0,0 +1,40 @@
+# Windows Container Workloads Prerequisites
+Below are the prerequisites for Windows Container Workloads. Please also see the cloud provider-specific sections that 
+may be relevant.
+
+## Supported Kubernetes Distribution Version
+* OKD/OCP 4.6+ must be used in order to configure Windows VMs
+* OKD/OCP 4.8+ and WMCO 3.0+ must be used in order to configure Bring-Your-Own-Host(BYOH) Windows instances
+
+## Supported Cloud Providers
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+## Supported Windows Server versions
+The table below outlines the supported 
+[Windows Server version](https://docs.microsoft.com/en-us/windows/release-health/release-information) based on the cloud
+platform you are using. *These are the only Windows Server versions supported by Windows Containers.* Any other Windows 
+Server version not listed means it is NOT currently supported by Win-C, and will give you errors. Please only use the 
+appropriate version according to the Cloud Provider you are using. 
+
+| *Cloud Provider*    | *Supported Windows Server version*                                      |
+| -----------         | -----------                                                             |
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): *Windows Server 2019*|
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): *Windows Server 2019*|
+| VMware vSphere      | Windows Server Semi-Annual Channel (SAC): *Windows Server 2004*         |
+
+## Supported Networking
+* [*OVNKubernetes* hybrid networking](docs/setup-hybrid-OVNKubernetes-cluster.md) is used for Window Containers. This is
```

Comment:
I think the link here is malformed -- might need to drop the `docs/` part as we are in that directory already